engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -544,9 +544,9 @@ class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
       .getAttribute(osBean.getObjectName, "TotalPhysicalMemorySize")
       .asInstanceOf[Long]
     val totalMemMB = totalMem / 1024 / 1024
-    ManagementFactory.getMemoryMXBean.getHeapMemoryUsage
     telemetryLog.trace(
-      "Initializing main module of the Language Server: edition={}, graal_version={}, enso_version={}, is_release={}, AOT={}, os_name={}, os_arch={}, os_version={}, available_cpus={}, total_memory_MB={}, available_memory_MB={}",
+      "Initializing main module of the Language Server: project_id={}, edition={}, graal_version={}, enso_version={}, is_release={}, AOT={}, os_name={}, os_arch={}, os_version={}, available_cpus={}, total_memory_MB={}, available_memory_MB={}",
+      serverConfig.projectId,
```

Comment:
- If `serverConfig.projectId` contains the project ID, then this would be a fix.
- I assume the change is backward compatible as ...
- ...telemetry will add `{ ... project_id : "xyz" ...}` to the JSON object being sent 
- thus old messages will not have `project_id` field
- and new messages will have it

Reply:
Exactly. There will only be new field - `metafield.project_id` in the telemetry message. All the other fields will still be there. I believe this can simply be handled by various ElasticSearch queries.